ext/native-decls/GetPedDecorations.md

```diff
@@ -0,0 +1,24 @@
+---
+ns: CFX
+apiset: client
+---
+## GET_PED_DECORATIONS
+
+```c
+object GET_PED_DECORATIONS(Ped ped);
+```
+
+Returns a list of decorations applied to a ped.
+
+The data returned adheres to the following layout:
+```
+[{collectionHash1, overlayHash1}, ..., {collectionHashN, overlayHashN}]
```

Comment:
`{}` would refer to an object, which this isn't.

Reply:
Monkey see, monkey do. I used existing FiveM documentation as a reference. But can change up.

https://github.com/citizenfx/fivem/commit/bc2f29bbc57da75a423c77fb760380f230b07b24